src/SFA.DAS.AssessorService.Application.Api/StartupConfiguration/Startup.cs

```diff
@@ -55,6 +55,12 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
             {
                 //services.AddApplicationInsightsTelemetry();
 
+                services.AddMvc()
```

Comment:
Adding all the Mvc() stuff into the pipeline seems a bit heavy handed to fix a json loop.  Anything else you can do to fix without dragging all that stuff in?

Reply:
sorry dave I can't see one that can quickly be dealt with. If you can see another way by all means let me know. I don't think is heavy handed is just an argument to the serialiser to allow it to deal with recursive loops - the real problem cause by exposing entity and code first - we could change the definition to expose a view model ? this would mean some extra work